### PR TITLE
Update log_collection.md

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?

Clarifies `build.gradle` file location as there is a top-level and module level build.gradle file

### Motivation

Jira request

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

**Review by jtappa — APPROVED** [review]

File: docs/log_collection.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
 
 ## Setup
 
-1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:
+1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file located in the root project directory:
```
Comment (0xnm) [inline docs/log_collection.md:13]: why this change is needed? this code block should be added to the particular module's `build.gradle`, not to the root (project) one.

**Review by mariusc83 — APPROVED** [review]

File: docs/log_collection.md (hunk 1 above)
Comment (sarina-dd) [reply docs/log_collection.md:13]: I can update to say the particular module's `build.gradle`. A customer gave feedback that they weren't sure which we were referencing.

Comment (sarina-dd) [inline docs/log_collection.md:13]:
```suggestion
1. Add the Gradle dependency by declaring the library as a dependency in the module-level `build.gradle` file:
```

**Comment by sarina-dd** [comment]
@0xnm Updated the PR! Let me know if we're good to merge now.

**Review by 0xnm — APPROVED** [review]
